lib/src/main/java/fi/epicbot/toster/model/Config.kt

```diff
@@ -22,6 +22,7 @@ class Config(
     var permissions: Permissions = Permissions(),
     var testTimeoutMillis: Long = 600 * 1000L,
     var deleteAndInstallApk: Boolean = true,
+    var swipeOffset: SwipeOffset = SwipeOffset(offsetPx = 220, offsetFactor = 0.08)
```

Comment:
Why is 220?

Reply:
just values from my head
verified for instagram :)